transform-test-selector-params-to-hash-pairs.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-env node */
+var TEST_SELECTOR_PREFIX = /data-test-.*/;
```

Comment:
yes, but I'd like to do that in a separate PR since we already have quite a bit of duplication around that regex